main.js

```diff
@@ -16,6 +16,7 @@ import { Provider } from "react-redux";
 import "@patternfly/patternfly/patternfly-addons.css";
 import "@patternfly/patternfly/layouts/Flex/flex.css";
 import "@patternfly/patternfly/utilities/Display/display.css";
+import "@patternfly/react-core/dist/styles/base.css";
```

Comment:
How is this change related to PF4 version upgrade?

Reply:
This is from the instructions provided by the `react-core` npm package [here](https://www.npmjs.com/package/@patternfly/react-core):

>  In order to have consistent styles throughout the application, PatternFly Base CSS should be used in the project, or some components may diverge in appearance. This import statement should be placed before your main application component is imported.
> 
> `import '@patternfly/react-core/dist/styles/base.css';`

Reply:
Fixed